src/components/Movies.vue

```diff
@@ -57,35 +70,35 @@ export default {
   name: "Movies",
   data() {
     return {
-      headers: [
-        {
-          sortable: false,
-          text: "#",
-          value: "movie.id"
-        },
-        {
-          sortable: false,
-          text: "Title",
-          value: "movie.title"
-        }
-      ],
       error: null,
-      search: ""
+      search: "",
+      genre: ""
     };
   },
   computed: {
     movies() {
+      if (this.genre.length > 0 && this.genre !== "all") {
+        const movies = this.$store.getters["movies/movies"].filter(
+          movie => movie.genre.name === this.genre
+        );
+        return movies;
+      }
       return this.$store.getters["movies/movies"];
     },
     pagination() {
       return this.$store.getters["movies/pagination"];
     },
     authenticated() {
       return this.$store.getters["user/isLoggedIn"];
+    },
+    genres() {
+      const genres = ["all", ...this.$store.getters["movies/genres"]];
```

Comment:
there is no need to create variable to keep result, we can return it directly. What is purpose of "all"?